apps/integration/src/app/integrations/flyout-tabs/flyout-tabs.module.ts

```diff
@@ -0,0 +1,15 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { SkyFlyoutModule } from '@skyux/flyout';
+import { SkyTabsModule } from '@skyux/tabs';
+
+import { FlyoutTabsContentComponent } from './flyout-tabs-content.component';
+import { FlyoutTabsComponent } from './flyout-tabs.component';
+
+@NgModule({
+  imports: [CommonModule, SkyFlyoutModule, SkyTabsModule],
+  declarations: [FlyoutTabsComponent, FlyoutTabsContentComponent],
+  entryComponents: [FlyoutTabsContentComponent],
```

Comment:
You shouldn't need `entryComponents` in Angular 12

Reply:
The `code-examples` SPA has tons of `entryComponents` set in them. Should I remove all of those while I'm at it?

Reply:
I would leave them for now and do that separately. We can file a bug for it.

Reply:
I'll do that.

Reply:
https://blackbaud.visualstudio.com/Products/_workitems/edit/2105675

Done!^